blog/11-streampark-usercase-tianyancha.md

```diff
@@ -0,0 +1,198 @@
+---
+slug: streampark-usercase-tianyancha
+title: Apache StreamPark™ helps Tianyancha real-time platform construction｜Multiple-fold increase in efficiency
+tags: [StreamPark, 生产实践]
+---
+
+![](/blog/tianyancha/new_cover.png)
```

Comment:
This snippet in the English version would still render Chinese.

Reply:
My bad. I missed this one